e2e/e2etests/e2etests.go

```diff
@@ -286,7 +286,7 @@ var (
 )
 
 // v2ZetaVersion is the minimum version that supports ZETA transfers using the gateway (v2 protocol contracts)
-const v2ZetaVersion = "v37.0.0"
+const v2ZetaVersion = "v38.0.0"
```

Comment:
This change should not be necessary, using `v37.0.0` is already sufficient to skip the tests in the first run